### PR TITLE
Double field

### LINKED ISSUE #278: Switch compound fields to double fields
- Remove compound field module
- Update all fields to double fields
  -- field_faq
  --field_step_pre
  -- filed_step_post

### PR DESCRIPTION
**Pull request by angaither**
Add old compound fields back in as double fields
Fixes #278

### CODE REVIEW

**Comment by aaronschachter** [comment]
This looks good.  I added a comment to #278 about updating the preprocess_entity function to use the corresponding "first" and "second" values instead of "title" and "body".  Can you either add these to the current PR or open up another one in order to fully resolve #278 ?

**Comment by angaither** [comment]
@aaronschachter good call, updated the keys!

**Comment by aaronschachter** [comment]
:+1:

File: drupal-org.make [hunk 1]
```diff
@@ -17,6 +17,10 @@ projects[ctools][subdir] = "contrib"
 projects[date] = "2.6"
 projects[date][subdir] = "contrib"
 
+; Double Field
+projects[date] = "2.3"
+projects[date][subdir] = "contrib"
```
Comment (aaronschachter) [inline drupal-org.make:22]: Whoops.

**Pull request merged**